packages/core/src/interfaces/components.ts

```diff
@@ -141,3 +145,21 @@ export interface ZoomBarOptions {
 	 */
 	data?: Object[];
 }
+
+/**
+ * customize the ToolBar component
+ */
+export interface ToolBarOptions {
+	/**
+	 * a variable to handle zoom in ratio
+	 */
+	zoomRatio?: number;
```

Comment:
shouldn't this be part of the zoombar options? we probably will also need it for scroll to zoom functionality

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/717/commits/50a9137867775ff491405b4b22971d9968682fa8.